src/applications/disability-benefits/526EZ/config/form.js

```diff
@@ -1,10 +1,10 @@
 import _ from '../../../../platform/utilities/data';
 
-import { omitRequired } from '../../../common/schemaform/helpers';
+// import { omitRequired } from '../../../common/schemaform/helpers';
 
 import fullSchema526EZ from 'vets-json-schema/dist/21-526EZ-schema.json';
 // NOTE: Easier to run schema locally with hot reload for dev
-// import fullSchema526EZ from '/local/path/vets-json-schema/dist/21-526EZ-schema.json';
+// import fullSchema526EZ from '/Users/adhocteam/Sites/vets-json-schema/dist/21-526EZ-schema.json';
```

Comment:
Maybe this change doesn't make sense? :man_shrugging:

Reply:
That's not what was supposed to happen...